### PR TITLE
docs: Add a warning about Aggregator metadata

### PR DESCRIPTION
**Pull request by silversupreme**
## Summary
We've seen customers have issues with Aggregator nodes over-writing log tags if this transform is enabled. This will at least note to people that this isn't a desirable configuration, or to know what they're getting into.

Better verbiage is very welcome.

### CODE REVIEW

**Comment by silversupreme** [comment]
The verbiage as of this comment ends up being rendered like this:

Screen Shot 2021-12-08 at 3 39 04 PM

**Comment by spencergilbert** [comment]
My only note is that the `-` feels weird to me, I don't remember seeing that anywhere else (could be wrong).

File: website/cue/reference/components/transforms/aws_ec2_metadata.cue [hunk 1]
```diff
@@ -35,7 +35,11 @@ components: transforms: aws_ec2_metadata: {
 				""",
 		]
 		notices: []
-		warnings: []
+		warnings: [
+			"""
+				Do not enable this transform if you are running Vector in Aggregator mode- tags will be sourced from the Aggregator node's metadata server and will not reflect correctly on input data.
```
Comment (binarylogic) [inline website/cue/reference/components/transforms/aws_ec2_metadata.cue:40]:
```suggestion
				Do not enable this transform if you are running Vector as an Aggregator, tags will be sourced from the Aggregator node's metadata server and not the client's.
```

**Review by jszwedko — APPROVED** [review]
👍 I think this is a good addition; especially since a user was confused about it.

I could imagine cases where a user _would_ want to use this transform an aggregator to annotate events with some information about the aggregator they were handled by (like instance ID), but this is likely a minority case.

**Pull request merged**